### DISCUSSION TITLE
miniPaint organization?

### DISCUSSION THREAD

**Discussion by ghost** [viliusle/miniPaint#357] [Q&A] [answered]
Hello, folks!

I wonder if you've all thought about adding a miniPaint organization on GitHub. Have you all considered adding a miniPaint organization on GitHub?

**Comment by viliusle** [comment, marked as answer]
no, but I need to research more about github organizations.

**Reply by ghost** [reply]
> no, but I need to research more about github organizations.

thanks for feedback. With an organization, GitHub can have custom GitHub pages for the software: (url-before: `http://viliusle.github.io/miniPaint/`) and (url-after: `http://minipaint.github.io/`)
